sycl/test-e2e/Matrix/joint_matrix_query_default.cpp

```diff
@@ -39,7 +39,8 @@ void matrix_multiply(big_matrix<T1, NUM_ROWS_C, NUM_COLS_C> &C,
   size_t K = NUM_COLS_A;
   assert(NUM_ROWS_C == NUM_ROWS_A && NUM_COLS_A == NUM_ROWS_B * 4);
 
-  using myparams2 = tpu_params<tpu::amx, int8_t, int8_t, int>;
+  using myparams2 =
+      matrix_params<architecture::intel_cpu_spr, int8_t, int8_t, int>;
```

Comment:
should be sycl::ext::oneapi::experimental::architecture.
otherwise, errmsg is reported
error: use of undeclared identifier 'architecture'